x/cron/abci.go

```diff
@@ -0,0 +1,22 @@
+package cron
+
+import (
+	"github.com/dymensionxyz/rollapp-wasm/x/cron/keeper"
+	cronTypes "github.com/dymensionxyz/rollapp-wasm/x/cron/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
+	allContracts := k.GetAllContract(ctx)
```

Comment:
The loop through all contract here and execute msg can become the bottle neck of the chain if there are many contracts. It also could lead to some critical issue if we can't control the sudo msg. I suggest not follow this approach

Reply:
We specifically fetch and execute contracts that have been whitelisted within the module, ensuring that the process remains streamlined and doesn't pose a bottleneck to the chain's performance.